downstream/modules/platform/ref-OCP-system-requirements.adoc

```diff
@@ -0,0 +1,7 @@
+
+
+// [id="ref-OCP-system-requirements_{context}"]
+
+= System requirements for installing {PlatformName} on {OCPShort}
+
```

Comment:
I wonder if you shouldn't make the titles here align with the titles of the different install guides? Suggest:
System requirements for installing on {OCPShort}

Reply:
Change made, thank you!